examples/demo_copy.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
```

Comment:
Might need to add the copyright header here and on the other example

Reply:
Actually these two examples were from the very beginning of gtc development. Not sure if they are even useful.

Reply:
I removed them rather than updating them and promisining maintenance. That should be a separate (mini-)project.